staging/nvidia/charts/grafana/templates/configmap-dashboards.yaml

```diff
@@ -0,0 +1,1125 @@
+# Generated from 'nodes' from https://raw.githubusercontent.com/coreos/kube-prometheus/master/manifests/grafana-dashboardDefinitions.yaml
+# Do not change in-place! In order to change this file first read following link:
+# https://github.com/helm/charts/tree/master/stable/prometheus-operator/hack
+{{- if .Values.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nvidia-grafana-gpu-dashboard
```

Comment:
Best practice is to have this name generated by the template.

Reply:
The main purpose was to make it explicit and it follows the example of Istio config map.